includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1176,7 +1178,9 @@ public function get_onboarding_required_verification_information( string $countr
 		return $this->request(
 			$params,
 			self::ONBOARDING_API . '/required_verification_information',
-			self::GET
+			self::GET,
+			true,
```

Comment:
maybe it would be a good idea to add a test that checks that the method was called with user_token = true, won't be easy to break it again :)

Reply:
Addressed in 0df59ed